src/otx/v2/adapters/torch/lightning/anomalib/engine.py

```diff
@@ -38,16 +39,16 @@ class AnomalibEngine(LightningEngine):
 
     def __init__(
         self,
+        task: TaskType,
```

Comment:
I'm not sure if having an Enum called TaskType is really convenient, because the user has to import the TaskType once to use the API.
What about taking a str as well and converting it?

Reply:
I agree with this.

Reply:
It's probably best to get both, and convert the str in the case of str.